tests/components/meteo_france/test_config_flow.py

```diff
@@ -57,32 +109,53 @@ async def test_user(hass, client_1):
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
     assert result["step_id"] == "user"
 
-    # test with all provided
+    # test with all provided with search returning only 1 place
     result = await hass.config_entries.flow.async_init(
         DOMAIN, context={"source": SOURCE_USER}, data={CONF_CITY: CITY_1_POSTAL},
     )
     assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
-    assert result["result"].unique_id == CITY_1_NAME
-    assert result["title"] == CITY_1_NAME
-    assert result["data"][CONF_CITY] == CITY_1_POSTAL
+    assert result["result"].unique_id == f"{CITY_1_LAT}, {CITY_1_LON}"
+    assert (
+        result["title"]
+        == f"{CITY_1_NAME} - {CITY_1_ADMIN} ({CITY_1_ADMIN2}) - {CITY_1_COUNTRY}"
+    )
+    assert result["data"][CONF_LATITUDE] == str(CITY_1_LAT)
+    assert result["data"][CONF_LONGITUDE] == str(CITY_1_LON)
+
+
+async def test_user_list(hass, client_2):
+    """Test user config."""
+
+    # test with all provided with search returning more than 1 place
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}, data={CONF_CITY: CITY_2_NAME},
+    )
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result["step_id"] == "cities"
```

Comment:
continue the test until the end of the config: `assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY ...`

Reply:
done